app/core/modals/Modal.tsx

```diff
@@ -8,6 +8,7 @@ export const Modal = ({
   title,
   body,
   onSubmit,
+  onCancel,
```

Comment:
This addition breaks the use of the modal component elsewhere.

Whenever you change an upstream function that is used elsewhere, it needs to be tested in all downstream instances. When you make a change like this, I would recommend checking where else this component/function is used to see whether it works there.

Reply:
Thanks for the catch! I'm updating the Modal, so that `onCancel` is an empty function. It looks like it's working on the draft page.